docs/coding-guidelines.md

```diff
@@ -333,6 +333,29 @@ The person writing the code will write the doc comments. Public APIs only. No ne
 
 Note: Public means callable by a customer, so it includes protected APIs. However, some public APIs might still be "for internal use only" but need to be public for technical reasons. We will still have doc comments for these APIs but they will be documented as appropriate.
 
+- Do not include empty XML comments.
+  - summary element must not be empty.
+  - param element must not be empty.
+
+Correct:
+
+```cs
+        /// <summary>
+        /// Calculates all fruit types
+        /// </summary>
+        /// <returns>A set of fruit types contain in this basket.</returns>
+        public ISet<FruitType> GetAllFruitTypes()
```

Comment:
How about an argument? `string fruitName`

Then the comment block examples of:
-  I don't want to mention any specifics (ie, what's the minimum acceptable text beyond empty string?)
- I want to mention only pre/post conditions
- I want to write a very detailed description.

Reply:
I updated it again, ptal.

I'm trying not to make this description too long. 
Any suggestions? 

I think the guidance is as simple as don't declare elements that are empty since the intellisense ignores them and they provide no tangible benefit.

Reply:
I see in your update that you want the elements, but with a basic description. That's exactly what I asked :) sounds good

Reply:
Cool. I added an extra example to illustrate that omitting obvious params is ok.